app/models/pillar.rb

```diff
@@ -2,7 +2,7 @@
 # rubocop:disable Metrics/ClassLength
 class Pillar < ApplicationRecord
   validates :pillar, presence: true
-  validates :value, presence: true, format: { with: /\A[\S]+(?: \S+)*\z/ }
+  validates :value, presence: true
```

Comment:
Not really, because on those very simple cases, the change here will silently fix those typos: https://github.com/kubic-project/velum/pull/532/files#diff-f5c24c314b640ac5d412bee1a158749dR135

Reply:
Having that regexp didn't allow us to have multi-line input fields (they didn't match the regexp), and since that regexp was merely checking for whitespaces at the beginning and at the end, why not strip those directly instead of failing a validation?

Reply:
Oh, gotcha. I didn't see the strip line. :sweat_smile: